tests/SprykerEcoTest/Zed/Braintree/Presentation/CreditCardGuestRefundCest.php

```diff
@@ -1,47 +0,0 @@
-<?php
```

Comment:
is there a reason to delete these tests?

Reply:
Reverted.